capstone-overview/intro.md

```diff
@@ -0,0 +1,159 @@
+# Capstone Project
+
+The Capstone Project is the largest undertaking you'll experience in the Ada classroom. Not only is it a month long, but you will be your own product owner. It will be up to you to decide (with input from instructors, mentors, and TAs) what gets built when (and how).
+
+## Learning Goals
+
+In the capstone project we will...
+
+- Demonstrate self-direction, time management, and independent learning
+- Learn and implement new technologies
+- Complete a product lifecycle from conception to delivery
+- Utilize agile practices learned to assist in project completion
+
+## Important Dates
```

Comment:
In my mind this would be a good place to note the uniqueness of the week of 1/25 as well, that it'll be primarily research/learning focused (with no PMing) and there'll be a deliverable at the end.